web/src/components/dashboard/SidebarCardsList.tsx

```diff
@@ -99,12 +99,23 @@ export const SidebarCardsList = (props: Props): ReactElement => {
     }
   };
 
+  const openInNewTab = (url: string): void => {
```

Comment:
[sand] maybe could be in helpers instead of just in this component?

Reply:
I'll move it to helpers for this ticket and fix our other instances to point to that in a seperate refactor ticket.